vulyk/blueprints/gamification/__init__.py

```diff
@@ -245,3 +250,12 @@ def donate() -> flask.Response:
             result={'done': False},
             errors=['Something wrong happened'],
             status=utils.HTTPStatus.BAD_REQUEST)
+
+
+def get_stats():
```

Comment:
Don't really like having this pimple on top level. May you just pass it as a bare lambda into `add_context_filler`?
Also the key `"stats"` is a bit confusing as I'd expect it to be a container having some data in it rather than a service.